File: Data Structures & Algorithms/search-2d-matrix/submission-1.swift
class Solution {
    func searchMatrix(_ matrix: [[Int]], _ target: Int) -> Bool {
        // brute force
        // O(n^2)
        for i in 0..<matrix.count {
            for j in 0..<matrix[0].count{
                if matrix[i][j] == target {
                    return true
                }
            }
        }
        return false 
    }
}

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.py
class Solution:
    def minSubArrayLen(self, target: int, nums: List[int]) -> int:
        # brute force 
         
        minLen = float('inf') # very large 

        for i in range(len(nums)): 
            runSum = 0
            for j in range(i, len(nums)):
                runSum += nums[j]

                if runSum >= target:
                    length = j - i + 1
                    minLen = min(length, minLen)
                    break
        if minLen == float('inf'):
            return 0
        return minLen 
                




        